charts/bitbucket-runner-autoscaler/templates/cleaner.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: bitbucket-runner-controller-cleaner
+  namespace: {{ .Values.namespace }}
+  labels:
```

Comment:
I would like to add common labels we tend to use, for example https://github.com/lablabs/dragonflydb-helm-chart/blob/872f3aeca968004ec4b783c10ef8a47239e3990e/charts/dragonfly/templates/_helpers.tpl#L34